splinterd/Cargo.toml

```diff
@@ -152,6 +154,9 @@ service-arg-validation = [
 ]
 service-endpoint = []
 ws-transport = ["splinter/ws-transport"]
+log-config = ["log-redirect","log-filter"]
+log-redirect = []
```

Comment:
log-filter and log-redirect are not used in this PR - please remove.

Reply:
fixed